### ISSUE TITLE
Add text color plugin

### ISSUE THREAD

**Issue by madc** [dimitrov-adrian/directus-extension-editorjs-interface#40]
I stumbled across this color plugin and thought it might be a valuable addition:
https://github.com/flaming-cl/editorjs-text-color-plugin

![Screenshot](https://user-images.githubusercontent.com/51183663/85190761-28853300-b2ee-11ea-9c11-13c3dbdef842.png)

Let me know, what you think.

**Comment by starsang1995** [comment]
Yes this plugin looks pretty awesome. Is this compatible with Directus?
